src/midi-router.js

```diff
@@ -9,7 +9,7 @@
 "use strict";
 
 var fluid = require("infusion"),
-    flock = fluid.require("flocking");
+    flock = fluid.require("flocking"); // eslint-disable-line no-unused-vars
```

Comment:
You're right, I was in the habit of doing it after various experiences with non-fluid global variables in IoC references.  It's fixed now.